src/s390x/zipl.rs

```diff
@@ -28,8 +28,11 @@ use tempfile::{Builder, NamedTempFile};
 
 /// Sets the boot device to `dev` using `chreipl`.
```

Comment:
Could we add a comment here to explain why we only want to do this on zvm? As someone who doesn't know anything about s390x it would probably help me understand. What are the other platfomrs and what will `systemd-detect-virt` return on s390x that isn't zVM?